### ISSUE TITLE
Can you repeat moco_v2 800epoch results(71.1) with multi-nodes(8 nodes, each nodes with 8 gpus)

### ISSUE THREAD

**Issue by haohang96** [HobbitLong/PyContrast#11]
Moco v2 report 800epoch results with batch-size 256. But if we train moco 800 epoch with only one node, it is very time-consuming. So have you re-run moco v2 with multi-nodes？

I noticed that you shuffle bn in each local node. I also implement a local shuffle bn before like your implementation. But I can not make sure in multi-nodes scenario, if there will be performance dropping by using linear lr&batch_size strategy  and local shuffle bn strategy?

**Comment by HobbitLong** [comment]
Hi, @haohang96,

I haven't trained MoCo v2 with 64 GPUs. `local shuffle` will not affect the performance, as in theory it's the same as original MoCo (e.g., shuffling 256 examples withing 8 GPUs). 

`linear lr&batch_size` might affect it, but I did not look closely at it.

**Comment by haohang96** [comment]
I understant that local shuffle will not affect the performace.  I still wonder what is your configuration when you run 800epoch experiments (how many gpus are used, if more than 8 gpus, what is the lr&batch_size)? 

If only 8gpus are used, 800epoch experiments will take a very long time....   If you face the same problem and how you deal with it?  Thanks !

**Comment by HobbitLong** [comment]
Hi, @haohang96,

For ResNet-50, I trained InfoMin with 8 GPUs for epochs, with lr as 0.03 and a batch size of 256. It will take 2 weeks but I only need to run it for once, so it's not a big issue once the parameters have been selected with a lower-epoch schedule.

For the setting of other models, please refer to [model_zoo](https://github.com/HobbitLong/PyContrast/blob/master/pycontrast/docs/MODEL_ZOO.md). Learning rate will be linearly scaled according to batch size.
